server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestOutlineTreeLineNumbers.java

```diff
@@ -49,12 +50,13 @@ void test() {
     Map<String, LineRange> expectedRanges =
         new ImmutableMap.Builder<String, LineRange>()
             .put("COPY BAR", new LineRange(0, 0))
-            .put("PROGRAM", new LineRange(1, 5))
-            .put("IDENTIFICATION DIVISION", new LineRange(1, 1))
-            .put("DATA DIVISION", new LineRange(2, 5))
-            .put("WORKING-STORAGE SECTION", new LineRange(3, 5))
-            .put("STRUCTNAME", new LineRange(4, 5))
-            .put("VARNAME", new LineRange(5, 5))
+            .put("PROGRAM: TEST", new LineRange(1, 6))
```

Comment:
new result:
![image (3)](https://user-images.githubusercontent.com/69206564/228207809-5fefb288-558a-4673-bd78-fe2cb500b71a.png)

Reply:
So, added a program-id statement to get the outline